packages/block-library/src/post-title/block.json

```
@@ -38,10 +38,15 @@
 		"typography": {
 			"fontSize": true,
 			"lineHeight": true,
-			"__experimentalFontFamily": true,
-			"__experimentalFontWeight": true,
 			"__experimentalFontStyle": true,
-			"__experimentalTextTransform": true
+			"__experimentalFontWeight": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalTextTransform": true,
+			"__experimentalDefaultControls": {
+				"fontSize": true,
+				"fontAppearance": true,
+				"__experimentalTextTransform": true
```

No `__experimental` prefix is needed here. The text-transform control doesn't currently display as a default.

```suggestion
				"textTransform": true
```